olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationError.h

```diff
@@ -55,38 +54,11 @@ class AUTHENTICATION_API AuthenticationError {
   inline bool ShouldRetry() const { return is_retryable_; }
 
  private:
-  static client::ErrorCode GetErrorForNetworkErrorCode(int network_code) {
-    switch (network_code) {
-      case network::Network::ErrorCode::IOError:
-        return client::ErrorCode::NetworkConnection;
-
-      case network::Network::ErrorCode::InvalidURLError:
-        return client::ErrorCode::NotFound;
-
-      case network::Network::ErrorCode::Offline:
-        return client::ErrorCode::ServiceUnavailable;
-
-      case network::Network::ErrorCode::Cancelled:
-        return client::ErrorCode::Cancelled;
-
-      case network::Network::ErrorCode::AuthorizationError:
-      case network::Network::ErrorCode::AuthenticationError:
-        return client::ErrorCode::AccessDenied;
-
-      case network::Network::ErrorCode::TimedOut:
-        return client::ErrorCode::RequestTimeout;
-
-      case network::Network::ErrorCode::UnknownError:
-      default:
-        return client::ErrorCode::Unknown;
-    }
-  }
-
   static bool IsRetryableNetworkCode(int network_code) {
```

Comment:
I wonder if it is not better to adapt http::HttpStatusCode::IsRetryable() with the below values and use that instead of own method.

Reply:
Great suggestion, please add an improvement task for that